internal/utils/me_utils.go

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package util
+package utils
```

Comment:
nitpick: If this is deemed too small for it's own pkg, maybe monitoring entity ID, uuid and ext ID could be grouped in an id/idutils package.
It's a bit strange that UUID and extID get packages, while meID does not